privacyidea/lib/tokens/webauthntoken.py

```diff
@@ -1004,6 +1021,9 @@ def get_init_detail(self, params=None, user=None):
             if (public_key_credential_creation_options.get("extensions") or {}).get("authnSel"):
                 response_detail["webAuthnRegisterRequest"]["authenticatorSelectionList"] \
                     = public_key_credential_creation_options["extensions"]["authnSel"]
+            if public_key_credential_creation_options.get("excludeCredentials"):
+                response_detail["webAuthnRegisterRequest"]["excludeCredentials"] = \
+                    public_key_credential_creation_options.get("excludeCredentials")
```

Comment:
Was there a reason why we remove these entries instead of just sending an empty list?